src/Microsoft.Azure.AppConfiguration.AspNetCore/Microsoft.Azure.AppConfiguration.AspNetCore.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\..\build\NugetProperties.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.1;net6.0;net7.0</TargetFrameworks>
```

Comment:
Why did we remove netstandard2.0 here?

Reply:
https://github.com/microsoft/FeatureManagement-Dotnet/pull/267 I'm basing things off of this PR in the feature management repo.

Reply:
because asp.net core doesn't support netstandard 2.0